src/environment_setup/netmhc.ml

```diff
@@ -196,19 +227,24 @@ let default ~run_program ~host ~install_path ~(files:netmhc_file_locations) () =
     | Some v -> (int_of_string (Char.escaped v)) < 4 
     | None -> true
   in
+  let netmhc_env = guess_env_setup ~install_path files.netmhc in
   let older_netmhc =
     default_netmhc_install ~run_program ~host ~install_path
     ~tool_file_loc:files.netmhc ~binary_name:"netMHC"
     ~example_data_file:(Some "SLA-10401/bl50/synlist") 
-    ~env_setup:(guess_env_setup ~install_path files.netmhc)
+    ~env_setup:(
+      [ `GENERIC ("/usr/local/bin/python2.5", "`which python`") ]
```

Comment:
yeah, in the tcsh script. We could just replace it with the full path to the python that we get from the Conda, but this comes with some flexibility. Also: in the file where the script decides on what python to use, the other option is this one (so this is as it is for consistency).